### PR TITLE
update `information_schema.processlist` to correctly display status of processes and databases

### PR DESCRIPTION
**Pull request by JCOR11599**
We used to hardcode `"Query"`, now we reference `process.Command`
Additionally, we now get the database from the current session and use that variable.

fix for: https://github.com/dolthub/dolt/issues/6023

### CODE REVIEW

File: sql/information_schema/information_schema.go [hunk 1]
```diff
@@ -1082,7 +1082,7 @@ func processListRowIter(ctx *Context, c Catalog) (RowIter, error) {
 			proc.User,                    // user
 			ctx.Session.Client().Address, // host
 			db,                           // db
```
Comment (zachmu) [inline sql/information_schema/information_schema.go:1084]: Pretty sure this is wrong too: it's going to always return the current DB, not the one the process is connected to

Fix plz

**Review by zachmu — COMMENTED** [review]
Your fix is good but there are other bugs in the current implementation

File: sql/information_schema/information_schema.go (hunk 1 above)
Comment (zachmu) [reply sql/information_schema/information_schema.go:1084]: Also check out the `show processlist` implementation, I'm not positive that field is correct there either

Comment (JCOR11599) [reply sql/information_schema/information_schema.go]: You are right, we actually never set the `Database` var; just added the fix for this.

File: sql/information_schema/information_schema.go [hunk 2]
```diff
@@ -1063,29 +1063,30 @@ func processListRowIter(ctx *Context, c Catalog) (RowIter, error) {
...
 		}
-		if len(status) == 0 {
+		if len(status) == 0 && proc.Command == ProcessCommandQuery {
 			status = []string{"running"}
 		}
 		sort.Strings(status)
+
+		db := proc.Database
+		if db == "" {
+			db = "NULL"
```
Comment (zachmu) [inline sql/information_schema/information_schema.go:1078]: Should this be the string NULL or an actual null value

**Review by zachmu — APPROVED** [review]
LGTM

**Pull request merged**